app/src/integrationTests/ReportIntegrationTests.tsx

```diff
@@ -109,51 +94,57 @@ class ReportIntegrationTests extends IntegrationTestSuite {
         it("downloads artefact", async () => {
             const artefact = mockArtefact({filenames: ["all.csv", "all.png"], description: "all things"});
             const versions = await (new ReportsService(this.store.dispatch, this.store.getState)).getReportVersions("multi-artefact");
-            const rendered = shallow(<ArtefactItem report={"multi-artefact"} version={versions[0]}
-                                                   artefact={artefact}/>);
+            const rendered = sandbox.mount(<Provider store={this.store}><ArtefactItem report={"multi-artefact"} version={versions[0]}
```

Comment:
Testing the full mounted component (so implicitly also testing the `OneTimeLinkContext` here, which we weren't doing before.)